portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncSitesAndChannels.testcase

```diff
@@ -417,7 +423,7 @@ definition {
 		}
 
 		task ("Click next page") {
-			Click(locator1 = "ACDXPSettings#ASSIGN_MODAL_NEXT_PAGE");
+			ACUtils.checkAnyPage(pageNumber = 2);
```

Comment:
Hey @joshitagaki, this macro might not work in some cases. The path used in this macro will find two elements on the wizard step 2. This happens because when the modal is enabled the Path finds the pagination of the list of properties and the pagination of the modal (channels and sites). So if you have many properties in the list or you have modified the pagination of the list of properties the Path of these macros will not work in the modal since there are two paginations on the same page

So I think it's best to correct the Path that was used so that there are no cases of errors for this. Then you change it to `//div[contains(@class,'fade show')]//a[contains(@aria-label,'next')]` or `//div[contains(@class,'fade show')]//*[contains(@class,'page-link')][normalize-space(text())='${key_pageNumber}']` which will work for next and previous

What do you think?